tests/test_pickling.py

```diff
@@ -19,6 +19,7 @@ def test_roundtrip():
 
 
 def test_roundtrip_with_dict():
+    return False
```

Comment:
Hm, this test was being skipped but was reported as 'passed' because there were no assertion errors. Should be checked on PyPy.